src/vsg/vk/PhysicalDevice.cpp

```diff
@@ -30,12 +33,17 @@ PhysicalDevice::PhysicalDevice(Instance* instance, VkPhysicalDevice device, int
     _rayTracingProperties = {};
     _rayTracingProperties.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_RAY_TRACING_PROPERTIES_NV;
     _rayTracingProperties.pNext = nullptr;
-
-    // do we check for extension support?
-    VkPhysicalDeviceProperties2 deviceProps2;
-    deviceProps2.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_PROPERTIES_2;
-    deviceProps2.pNext = &_rayTracingProperties;
-    vkGetPhysicalDeviceProperties2(_device, &deviceProps2);
+    
+    if( VK_VERSION_MINOR(_properties.apiVersion) > 1  ||
+       ( VK_VERSION_MINOR(_properties.apiVersion) == 1 && VK_VERSION_MINOR(_properties.apiVersion) >= 2 ))
```

Comment:
Did you mean to use something different there?? As it looks like it will always evaluate to false....